docs/12-workflows/02-working-with-workflows/index.md

```diff
@@ -60,3 +64,17 @@ To add a node:
 ::: tip
 You can copy and paste nodes in a workflow or between workflows via the context menu or [kbd]Cmd/Ctrl + C[kbd] and [kbd]Cmd/Ctrl + V[kbd].
 :::
+
+### Sequential execution
```

Comment:
I moved this section to this page as it made more sense to follow after the nodes section